gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
Is it totally necessary to update gradle version for this PR?

Reply:
The kotlin plugin didn't seem to work on 3.2. However, I agree with you that we should update Gradle version in a different PR. Any reason why you're still on 3.2?

Reply:
Our release process and all plugins are tested on 3.2
Of course we can bump gradle version, but I should verify everything first.

Reply:
let me know if you need any help. My suggestion would be to just move to 4.10.3, it is the latest version available for the 4.x branch. However, I don't know how many breaking changes have been since 3.2